### PR TITLE
Administrator: migration from user key to master key on mysql

### PR DESCRIPTION
**Pull request by thommierother**
Tested migration fro user key to master key on mysql.

### CODE REVIEW

File: modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc [hunk 1]
```diff
@@ -1,34 +1,36 @@
-= Migrating User Key Encryption to Master Key Encryption
+= Migrating User Key-based Encryption to Master Key Encryption
 
-== Why Should I Move Away From User Key-based Encryption
+== Why Should I Move Away From User Key-based Encryption?
 
-NOTE: User key-based encryption is planned to be removed from ownCloud in the near future. For new installations, Master key-based encryption is the recommended setup. The decryption workflow described here will only work when users have enabled password recovery and if an admin recovery password is available for the ownCloud administrator.
+NOTE: User key-based encryption is planned to be removed from ownCloud in the near future. While it is a bit more secure than a central encryption approach, User key-based encryption has some disadvantages. It blocks some additional functions such as the integration of an online editor like LibreOffice or OnlyOffice into ownCloud and can cause problems when sharing files with groups. Therefore Master-key-based encryption is now the recommended setup fr all new insallations.
```
Comment (settermjd) [inline modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc]: Please split each sentence onto a separate line. Secondly, in the last sentence, it should end with "recommended setup for all new installations".

File: modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
 ----
-sudo -u www-data occ encryption:decrypt-all --continue=yes && \
-  sudo -u www-data occ encryption:disable --no-interaction && \
-  sudo -u www-data occ app:disable --no-interaction encryption
+sudo -u www-data php occ encryption:decrypt-all --continue=yes && \
+  sudo -u www-data php occ encryption:disable --no-interaction && \
+  sudo -u www-data php occ app:disable --no-interaction encryption
 ----
 
+NOTE: The decryption of the files by the central administration requires the current passwords of all users!
```
Comment (settermjd) [inline modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc]: Where you've written "central administration", did you mean the systems administrator or ownCloud administrator?

File: package.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
   ],
   "dependencies": {
     "@antora/cli": "^1.1.1",
-    "@antora/site-generator-default": "^1.1.1"
+    "@antora/site-generator-default": "^2.0.0"
```
Comment (settermjd) [inline package.json]: Why was the Antora version bumped in this PR? This should, ideally, be in a separate PR.

Comment (thommierother) [reply package.json]: ok, sorry, dondn't know that

File: modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc (hunk 2 above)
Comment (thommierother) [reply modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc]: the owncloud administrator

File: modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc (hunk 1 above)
Comment (cdamken) [reply modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc]: A typo in the last sentence "fr all new insallations." the `for`

File: modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc (hunk 2 above)
Comment (cdamken) [reply modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc]: I would add: only work when users have enabled password recovery and if an admin recovery password is available

File: modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc (hunk 1 above)
Comment (settermjd) [reply modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc]: Spotted that one and left a comment already.

File: package.json (hunk 3 above)
Comment (settermjd) [reply package.json]: All good. Just worth mentioning.

**Comment by settermjd** [comment]
Closing as #1422 completes the work.
